data/sql/mariadb_example.sql

```diff
@@ -0,0 +1,164 @@
+                                street CHARACTER VARYING(100),
+                                city CHARACTER VARYING(100),
+                                state CHARACTER VARYING(100),
+                                zip CHARACTER VARYING(100)
+);
+
+CREATE TABLE customer (
+                                 id INT PRIMARY KEY,
+                                 email CHARACTER VARYING(100),
+                                 name  CHARACTER VARYING(100),
+                                 created TIMESTAMP,
+                                 address_id BIGINT
+);
+
+CREATE TABLE employee (
+                                 id INT PRIMARY KEY,
+                                 email CHARACTER VARYING(100),
+                                 name CHARACTER VARYING(100),
+                                 address_id BIGINT
+);
+
+CREATE TABLE payment_card (
+                                     id CHARACTER VARYING(100) PRIMARY KEY,
+                                     name CHARACTER VARYING(100),
+                                     ccn BIGINT,
+                                     code SMALLINT,
+                                     preferred BOOLEAN,
+                                     customer_id INT,
+                                     billing_address_id BIGINT
+);
+
+CREATE TABLE orders (
+                               id CHARACTER VARYING(100) PRIMARY KEY,
+                               customer_id INT,
+                               shipping_address_id BIGINT,
+                               payment_card_id CHARACTER VARYING(100)
+);
+
+CREATE TABLE order_item (
+                                   order_id CHARACTER VARYING(100),
+                                   item_no SMALLINT,
+                                   product_id INT,
+                                   quantity SMALLINT
+);
+
+CREATE TABLE visit (
+                              email CHARACTER VARYING(100),
+                              last_visit TIMESTAMP
+);
+
+CREATE TABLE login (
+                              id INT PRIMARY KEY,
+                              customer_id INT,
+                              time TIMESTAMP
+);
+
+CREATE TABLE service_request (
+                                        id CHARACTER VARYING(100) PRIMARY KEY,
+                                        email CHARACTER VARYING(100),
+                                        alt_email CHARACTER VARYING(100),
+                                        opened DATE,
+                                        closed DATE,
+                                        employee_id INT
+);
+
+CREATE TABLE report (
+                               id INT PRIMARY KEY,
+                               email CHARACTER VARYING(100),
+                               name CHARACTER VARYING(100),
+                               year INT,
+                               month INT,
+                               total_visits INT
+);
+
+-- Populate tables with some public data
+INSERT INTO product VALUES
```

Comment:
To save me doing this as part of https://github.com/ethyca/fidesops/pull/185, it would be fantastic if you could put this initial data seeding into a fixture that can be run before / after each test. In a similar way to either:
- https://github.com/ethyca/fidesops/pull/185/files#diff-388421657341e3fb1e43009b6b8472f13d35b2a873516dfae800f7d2d2c11ee0R19, or
- https://github.com/ethyca/fidesops/blob/main/tests/service/privacy_request/request_runner_service_test.py#L576

Reply:
ok, I'll give that a try!